package.json

```diff
@@ -222,11 +223,16 @@
   },
   "pre-commit": {
     "run": [
-      "lint",
```

Comment:
by removing this I think the linter does not work during rebasing and it complains there is an error, which makes hard to find out which commit introduced the error, and the fix requires another commit unless you are willing to detect that commit. Sorry if I am wrong but I would like to propose to get this back.